src/protocols/device_path.rs

```diff
@@ -52,3 +52,20 @@ impl Hardware {
     pub const SUBTYPE_CONTROLLER: u8 = 0x05;
     pub const SUBTYPE_BMC: u8 = 0x06;
 }
+
+#[repr(C, packed)]
+pub struct HardDriveDevicePathNode {
```

Comment:
I think I would call it `HardDriveMedia`? This would follow the naming scheme we already have in that file, where we drop `DevicePath` suffixes and use the Table-names from `10.3` in the specification.

Reply:
Agree and done